### PR TITLE
Feed: Do not include empty parts in the response

### PR DESCRIPTION
**Pull request by d00rman**
Bug: [T144990](https://phabricator.wikimedia.org/T144990)

### CODE REVIEW

**Comment by d00rman** [comment]
cc @wikimedia/services @berndsi @mdholloway

**Comment by mdholloway** [comment]
👍

File: v1/feed.js [hunk 1]
```diff
@@ -120,7 +120,9 @@ Feed.prototype.aggregated = function(hyper, req) {
             };
             // populate its body
             Object.keys(result).forEach(function(key) {
-                finalResult.body[key] = result[key].body;
+                if (!!result[key].body && Object.keys(result[key].body).length) {
```
Comment (gwicke) [inline v1/feed.js]: Minor: The `!!` seems to be unnecessary here, and might be less clear than a plain "truthy" test.

Comment (d00rman) [inline v1/feed.js]: {{done}}

**Pull request merged**